web/src/pages/apd/new/ApdNew.js

```diff
@@ -0,0 +1,585 @@
+                                      value: 'asNeededUpdate',
+                                      checked: typeStatus.asNeededUpdate
+                                    }
+                                  ]}
+                                  labelClassName="ds-u-margin-bottom--1"
+                                  type="checkbox"
+                                  onChange={({ target: { value } }) => {
+                                    typeStatus[value] = !typeStatus[value];
+                                    onChange(
+                                      Object.keys(typeStatus).filter(
+                                        key => typeStatus[key]
+                                      )
+                                    );
+                                    setValue(
+                                      'updateStatus.typeStatus',
+                                      typeStatus,
+                                      { shouldValidate: true }
+                                    );
+                                  }}
+                                  onBlur={onBlur}
+                                  onComponentBlur={onBlur}
+                                  errorMessage={
+                                    errors?.updateStatus?.updateList?.message
+                                  }
+                                  errorPlacement="bottom"
+                                />
+                              )}
+                            />
+                          </div>
+                        )
+                      },
+                      {
+                        label: 'No, this is for a new project.',
+                        value: 'no',
+                        checked: value === 'no'
+                      }
+                    ]}
+                    onChange={e => {
+                      onChange(e);
+                    }}
+                    onBlur={onBlur}
+                    onComponentBlur={onBlur}
+                    errorMessage={errors?.typeStatus?.mmisUpdate?.message}
+                    errorPlacement="bottom"
+                  />
+                )}
+              />
+
+              <Controller
+                name="apdOverview.medicaidBA"
+                control={control}
+                render={({ field: { onBlur, onChange } }) => (
+                  <ChoiceList
+                    label="Medicaid Business Areas"
+                    hint={
+                      <div>
+                        Select the Medicaid Enterprise Systems Business Area(s)
+                        that cover the scope of this APD. A more detailed
+                        description of these business areas, along with the
+                        associated outcomes and metrics, are available at the
+                        MES Certification Repository.
+                      </div>
+                    }
+                    type="checkbox"
+                    choices={[
```

Comment:
I was searching around. I think to do this without having to maintain and external list is to add Controllers around each choice. I think we could make a new component called ControlledChoice that creates a controller and then uses the Choice component in the web/src/components/Choice.